src/native_main.nim

```diff
@@ -152,15 +182,48 @@ proc handleMessage(msg: MessageRecv): string =
                 reply.code = some(2)
 
         of "move":
-            let
-                dest = expandTilde(msg.to.get())
-                src  = expandTilde(msg.`from`.get())
-            if fileExists(dest):
+            let src = expandTilde(msg.from.get())
+            let dst = expandTilde(msg.to.get())
```

Comment:
`from` is a reserved word in Nim in some contexts. It doesn't matter here but for consistency I'd prefer the backticks remained.

(The backticks allow you to use reserved names as variable names)